src/components/FilterModal/FilterModal.jsx

```diff
@@ -0,0 +1,40 @@
+import "./filterModal.css";
+
+export function FilterModal({setIsFilterVisible}){
+    return (
+        <div className="modal-wrapper">
+            <output className="modal filter-modal">
```

Comment:
Great, got to know about a new tag. But on searching about it, I found that it is used to show the result of a calculation.

So, using it here would be good? As I am guessing, that using it you are only showing the modal for the filter, there is no calculation here, so using it here is a good choice?

Reply:
I used this because it is good for accessiblity. We can use it for output of any calculation or response to any user action.

I found about this tag through this article - https://web.dev/building-a-toast-component/#markup

Reply:
oh! Great then. Thanks for sharing the article link, will definitely give it a read. 
And awesome work man,